package com.resturante.logica.repositories;


import com.resturante.logica.models.Plato;
import org.springframework.data.jpa.repository.JpaRepository;

public interface PlatoRepository extends JpaRepository<Plato,Long> {
}
